DESCRIPTION

```diff
@@ -51,6 +50,8 @@ Suggests:
     raster,
     rmarkdown,
     sf,
+    silicate,
```

Comment:
```suggestion
```
We don't actually use an silicate functions at all, so that package itself is not needed. The `osmdata_sc` function merely produces output in a format compatible with silicate functionality. Once that's removed, I'll happily approve and merge.

Reply:
And FYI that funciton was developed purely to provide the best input for [the `dodgr` package](https://github.com/atfutures/dodgr).